packages/contracts/src/dollar/core/CreditClock.sol

```diff
@@ -5,31 +5,36 @@ import "abdk/ABDKMathQuad.sol";
 import {IAccessControl} from "../interfaces/IAccessControl.sol";
 import "../libraries/Constants.sol";
 
+/**
+ * @notice CreditClock contract
+ */
 contract CreditClock {
     using ABDKMathQuad for uint256;
     using ABDKMathQuad for bytes16;
 
-    // Manager contract.
+    /// @notice Access control contract
     IAccessControl public accessCtrl;
```

Comment:
```suggestion
    IAccessControl public accessControl;
```

I know that this is probably not the appropriate pull request to handle renames, but, this contraction seems unnecessary. I think we should not shorten variable names just to save a few keystrokes.

Reply:
I agree this readbility it's important in the whole codebase, sometimes it can be skipped to have a few shorten vars but not always

Reply:
https://github.com/ubiquity/ubiquity-dollar/issues/737